src/SoapCore/SoapCore.csproj

```diff
@@ -2,9 +2,9 @@
 
 	<PropertyGroup>
 		<Description>SOAP protocol middleware for ASP.NET Core</Description>
-		<Version>1.1.0.49</Version>
+		<Version>1.2.0.0</Version>
 		<Authors>Digital Design</Authors>
-		<TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+		<TargetFrameworks>netstandard2.0;netstandard2.1;net7.0;net8.0</TargetFrameworks>
```

Comment:
I don't see the benefit in removing net core 3.1, net 5 $ net. At the same time only marginal benefit of have net 7. I would propose we first merge in #1052

Reply:
I haven't had the time to work on this one for a while. Will probably resume work in a couple of weeks.
net core 3.1 and net 5 are already covered by targeting netstandard.
Agree about net 7 though. It's already out of support and I don't see a reason to keep it around

Reply:
Looking at the dependency graph if you want to reduce target framework I would stick with net core 3 as otherwise u would be forcing all projects < net 8 to take on a deprecated dependencies.

Reply:
does this works for you? we had some caveats making this work

Reply:
When will this be merged with the already merged pr's with a new nuget package?